src/contexts/SidebarContext.js

```diff
@@ -0,0 +1,13 @@
+import { createContext, useState, useContext } from "react";
+
+const SidebarContext = createContext();
+
+const SidebarProvider = ({ children }) => {
+  const [showFilterBar, setShowFilterBar] = useState(false);
+
+  return <SidebarContext.Provider value={{ showFilterBar, setShowFilterBar }}>{children}</SidebarContext.Provider>;
```

Comment:
suggestion: you are missing code format like prettier, configure in vscode to run on save of files or before commit

Reply:
I've saved this file with prettier, it's formatting like this only.